.cypress/integration/notebooks_test/notebooks.spec.js

```diff
@@ -135,7 +135,7 @@ describe('Testing paragraphs', () => {
     cy.get('a[data-test-subj="createNotebookPrimaryBtn"]').click();
     cy.get('input[data-test-subj="custom-input-modal-input"]').focus().type(TEST_NOTEBOOK);
     cy.get('button[data-test-subj="custom-input-modal-confirm-button"]').click();
-    cy.get('div[data-test-subj="notebookTitle"]').contains(TEST_NOTEBOOK).should('exist');
+    cy.get('h3.euiTitle').contains(TEST_NOTEBOOK).should('exist');
```

Comment:
any other placeholder to test this similar to a data-test-subj? might fail if the ui changes

Reply:
With moving the tittle from the div cypress was unable to find the data-test-subj attached to it. Will dig in deeper for a better solution when re-writing the test in follow-up PR.